design/vela-core/migrate-from-oam-runtime.md

```diff
@@ -0,0 +1,145 @@
+has already merged into [vela-core](https://github.com/oam-dev/kubevela/pull/663) now. Here is the doc for users who want
+to migrate from OAM Runtime to use KubeVela.
+
+## For Users Who are Using OAM runtime as Standalone Controller
+
+If you are using OAM Runtime as Standalone Controller, migrating to KubeVela is very straight forward.
+
+Server-side KubeVela(We call it `vela-core` for convenience) now include following **BUILT-IN** CRD and controllers.
+
+| Type |  CRD   | Controller  | From |
+| ---- |  ----  | ----  | ----  |
+| Framework | `applicationconfigurations.core.oam.dev` | Yes | OAM Runtime |
+| Framework | `components.core.oam.dev` | No | OAM Runtime |
+| Workload | `containerizedworklaods.core.oam.dev` | Yes | OAM Runtime |
+| Scope | `healthscope.core.oam.dev` | Yes | OAM Runtime |
+| Trait | `manualscalertraits.core.oam.dev` | Yes | OAM Runtime |
+| Framework | `scopedefinitions.core.oam.dev` | No | OAM Runtime |
+| Framework | `traitdefinitions.core.oam.dev` | No | OAM Runtime |
+| Framework | `workloaddefinitions.core.oam.dev` | No | OAM Runtime |
+| Trait | `autoscalers.standard.oam.dev` | Yes | New in KubeVela |
+| Trait | `metricstraits.standard.oam.dev` | Yes | New in KubeVela |
+| Workload | `podspecworkloads.standard.oam.dev` | Yes | New in KubeVela |
+| Trait | `route.standard.oam.dev` | Yes | New in KubeVela |
+
+CRDs and Controllers in the table from 'OAM Runtime' are exactly the same ones in `oam-kubernetes-runtime`.
+So in KubeVela we have added 4 more new CRDs with controller. 
+
+### Option 1: You want only Pure OAM Runtime by using vela-core with No additional traits and workloads.
+
+1. Find you deployment
+
+```shell script
+$ kubectl -n oam-system get deployment -l app.kubernetes.io/name=oam-kubernetes-runtime
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+oam-kubernetes-runtime-oam   1/1     1            1           62s
+```
+
+2. Update the deployment
+
+In this case, the deployment name of OAM runtime is `oam-kubernetes-runtime-oam`, let's edit it to update the image:
+
+```shell script
+$ kubectl -n oam-system edit deployment oam-kubernetes-runtime-oam
+```
+
+There are two changes:
+
+- update the image from `crossplane/oam-kubernetes-runtime:latest` to `oamdev/vela-core:latest`
+- add an args `- "--disable-caps=all"`, this will disable all additional workload and trait built in vela-core.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: oam-kubernetes-runtime-oam
+...
+      containers:
+        - name: oam
+          args:
+            - "--metrics-addr=:8080"
+            - "--enable-leader-election"
++           - "--disable-caps=all"
+-         image: crossplane/oam-kubernetes-runtime:latest
++         image: oamdev/vela-core:latest
+          imagePullPolicy: "Always"
+...
+```
+
+### Option 2: You want use fully functioned KubeVela
```

Comment:
Why don't we directly delete the old release and install KubeVela chart intead?

Reply:
That's the easiest way, and it couldn't not be called by "migrate"